docs/.vuepress/public/scripts/fixer.js

```diff
@@ -1,124 +1,126 @@
 /* eslint-disable */
```

Comment:
Why this file is changed? Can I ask you to clean up your PR and include only the files that you actually modified?

Reply:
> Why this file is changed? Can I ask you to clean up your PR and include only the files that you actually modified?

This PR contains only files that I changed. I change it to change all links to lower case. I was sure that the change in this file will change the links generated in the `API`.

Reply:
Still, if you have not modified the file it should not be listed in the "Files changed" tab. Can you sync the changes with the latest `feature/issue-7624` branch?

Reply:
I specially changed the `fixer.js` file because I adjusted the function `camelCase` into `lowerCase`. Update to the latest version of brancha `feature/issue-7624` is on f05c3e6375d9ebccbc09bdbb9644504920b473f4.

Reply:
But this file is not related to the PR at all. If I understand the [issue](https://github.com/handsontable/handsontable/issues/8330) correctly it's about changing the links in the Handsontable source code while API generation. The `fixer.js` is responsible only for demos while they are initialized.

Reply:
@budnix is right, your changes touch a completely different place that should.
The original issue describes incorrect links in JSDocs after from `./src/*` files - as shown in the issue's description.

![image](https://user-images.githubusercontent.com/2893099/123765936-7dc32f80-d8c6-11eb-9e2d-ebade21ea090.png)

Please look at the right side, where is a file name. This is the `*.md` file - generated from its JS source.
E.g.: `dropdownMenu.md` contains an incorrectly generated anchor to the `beforeInit` event.
The problem rather in jsdoc-converter for `{@link ....}` than in `fixer.js` - utility used to properly run examples for wrappers in JSFiddle.

Reply:
@budnix  you are right, I will revert these changes.

Reply:
@swistach Yes, exactly, @Siemienik works on this.